package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "eslint --config .eslintrc.js \"src/\" --ext .js,.jsx,.ts,.tsx && jest",
     "e2e-test-node": "jest --config=jest.e2e.config.js",
-    "e2e-test-browser": "testcafe",
+    "e2e-test-browser": "NODE_ENV=test testcafe",
```

Comment:
We could do this, but in my PR I did it in the code instead, which I think is a big cleaner (i.e., from the command line you can just run `testcafe` without having to think about the NODE_ENV value at all, but you can still override the 'test' value with an env var if you want):
```javascript
// Load environment variables from .env.test.local if available:
config({
  node_env: process.env.NODE_ENV || "test",
  path: __dirname,
  // In CI, actual environment variables will overwrite values from .env files.
  // We don't need warning messages in the logs for that:
  silent: process.env.CI === "true",
});
```

Reply:
That's better indeed. Turns out it even has a specific option for that: https://www.npmjs.com/package/dotenv-flow#optionsdefault_node_env

17e1776a